src/pages/projects/[pid]/document/[documentId]/vote/[id].js

```diff
@@ -49,66 +45,61 @@ export default function Index({project, document, change, changeVotes, userVoteP
 		return true;
 	}
 
-	const incrementVote = async () => {
-		if (!userCanVote()) {
-			return;
-		}
-		const result = await voteOnChange(change.cid, {vote: 1}, getCookie("stytch_session_jwt"));
-		if (!result) {
-			return;
-		}
-		if (userVote === 0) {
-			setTotalVotes(totalVotes + 1);
-		}
-		setUserVote(1);
-	};
-
-	const decrementVote = async () => {
-		if (!userCanVote()) {
-			return;
-		}
-		const result = await voteOnChange(change.cid, {vote: -1}, getCookie("stytch_session_jwt"));
-		if (!result) {
-			return;
-		}
-		if (userVote === 0) {
-			setTotalVotes(totalVotes + 1);
-		}
-		setUserVote(-1);
-	};
+	const ResultsSection = () => (
+		<>
+			<h2 className="mt-4 text-lg">Results</h2>
+			<AwaitResults/>
+		</>
+	);
```

Comment:
This should go in with the `AwaitResults` component, I suggest renaming it to `VoteResultsSection` and including the title inside of it. The component can then conditionally render either results or the fact that we are awaiting them. This would be the best way of structuring this behaviour.

Reply:
Ack() - really good feedback and idea